app/src/main/java/ch/epfl/sdp/appart/CreateUserActivity.java

```diff
@@ -61,17 +61,11 @@ public void createAccount(View view) {
             startActivity(intent);
         });
     }
-
     /**
-     * Method called when the Log In button is pushed
-     * Takes the user back to the login screen
-     *
-     * @param view the view that was clicked (i.e. the log in button)
+     * Closes the activity.
      */
-    public void backToLogin(View view) {
-        Intent intent = new Intent(this, LoginActivity.class);
-        startActivity(intent);
+    @Override
+    public void onBackPressed() {
+        finish();

```

Comment:
We can leave this override out, so that we don't have to test it

Reply:
Okay.